src/main/java/controller/LottoSystemController.java

```diff
@@ -0,0 +1,47 @@
+package controller;
+
+import domain.LottoSystem;
+import domain.Ranking;
+import domain.WinningNumbers;
+import domain.WinningResult;
+import java.util.Arrays;
+import view.InputView;
+import view.OutputView;
+
+public class LottoSystemController {
+
+    LottoSystem lottoSystem;
+    WinningResult winningResult;
+
+    public void run() {
+        buyLottoTickets();
+        decideWinningNumbers();
+        calculateResult();
+    }
+
+    private void buyLottoTickets() {
+        lottoSystem = LottoSystem.init(InputView.receivePrice());
+        OutputView.printNumberOfTickets(lottoSystem.getLottoQuantity());
+
+        OutputView.printLottoTickets(lottoSystem.getLottoTickets());
+    }
+
+    private void decideWinningNumbers() {
+        winningResult = lottoSystem.getWinningResult(
+                WinningNumbers.valueOf(
+                        InputView.receiveWinningNumbers(),
+                        InputView.receiveBonusNumber()
+                ));
+    }
+
+    private void calculateResult() {
+        OutputView.printRankResultTitle();
+        Arrays.stream(Ranking.values())
+                .filter(ranking -> ranking != Ranking.NOTHING)
```

Comment:
```suggestion
                .filter(ranking -> ranking.isNotNothing())
```
위와 같이 메세지를 던져서 구현해보는건 어떨까요 ^_^